### PR TITLE
Document the changes for 2022.06 release

### LINKED ISSUE #1165: update CHANGES.rst for release v2022.06
- [x] update `/CHANGES.rst`
- [x] update [wiki release page](https://github.com/nexusformat/definitions/wiki/releasenotes__v2022.06)

### PR DESCRIPTION
**Pull request by prjemian**
- close #1165

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,55 +1,94 @@
...
 Change History
 ##############
 
-:v2022.06:  expected *2022-06*
+v2022.06
+++++++++
+
+expected release *2022-07-01*
+
+This release is the result of two NeXus Code Camps ...
```
Comment (PeterC-DLS) [inline CHANGES.rst:25]: 2021-03 and 2022-06

**Comment by prjemian** [comment]
@nexusformat/developers Ready for review.  Final step before next release of definitions.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,55 +1,203 @@
...
+  * Andrea Albino
+
+* GitHub users
+
+  * cmmngr
+  * domna (Florian Dobener)
+
+* Helmholtz-Zentrum Berlin für Materialien und Energie
+
+  * Heike Görzig
```
Comment (prjemian) [inline CHANGES.rst:166]: NIAC member

File: CHANGES.rst [hunk 3]
```diff
@@ -1,55 +1,203 @@
...
+  * Sophie Hotz
+  * Kal Conley
+
+* Diamond
+
+  * Tim Snow
+
+* European Synchrotron Radiation Facility
+
+  * Wout de Nolf
```
Comment (prjemian) [inline CHANGES.rst:150]: NIAC member

File: CHANGES.rst [hunk 4]
```diff
@@ -1,55 +1,195 @@
...
+  * Tim Snow
+
+* FAIRmat
+
+  * Carola Emminger
+  * Florian Dobener
+  * Markus Kühbach
+  * Andrea Albino
+
+* GitHub users
```
Comment (prjemian) [inline CHANGES.rst:155]: included in FAIRmat team

File: CHANGES.rst [hunk 5]
```diff
@@ -1,55 +1,196 @@
...
+  ------
+
+Breaking Changes
+----------------
+
+* Included PDF files moved ``/pdf/`` to ``/_static/``.
+
+* Minimum Python requirement (for testing and building documentation):  3.7
+
+* ``NXlog``: Removed group named ``temperature``.
```
Comment (PeterC-DLS) [inline CHANGES.rst:57]: from `NXsample`

File: CHANGES.rst [hunk 6]
```diff
@@ -1,55 +1,196 @@
...
+  * Carola Emminger
+  * Florian Dobener
+  * Markus Kühbach
+  * Andrea Albino
+
+* National Synchrotron Light Source II
+
+  * Abby Giles
+  * Andi Barbour
+
```
Comment (PeterC-DLS) [inline CHANGES.rst:165]: Add explicit link to release note on wiki

**Comment by prjemian** [comment]
Ready for review.

**Review by PeterC-DLS — APPROVED** [review]
LGTM

**Pull request merged**